docker-compose-jupyter.yml

```diff
@@ -44,6 +44,8 @@ services:
         condition: service_healthy
     environment:
       AWS_REGION: us-west-2
+      AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
```

Comment:
We shouldn't need it with credential vending. The only credentials required from the Spark side are OAuth2 client id and client secret, unless credential vending doesn't work here.

Reply:
Yes, from the test, it doesn't seem to be working. It will throw an error as the AWS client key is not defined and not able to find needed auth to interact with s3.

Reply:
Can we invest why credential vending doesn't work? I assume it should be on by default.

Reply:
Sure thing. Will update shortly

Reply:
@flyrain so I tried to reproduce the issue again but fail to do so (not sure why it failed earlier). Credential vending is indeed working with 3 cleaned retries I just did. Close this one for now. Sorry for the confusion.